### PR TITLE
[ci skip] Docs: Clarify logs-vs-run output race

### PR DESCRIPTION
**Pull request by cevich**
See issue #2692

Signed-off-by: Chris Evich <cevich@redhat.com>

### CODE REVIEW

**Comment by cevich** [comment]
Anywhere else this/similar warning should be added?

File: docs/podman-logs.1.md [hunk 1]
```diff
@@ -7,9 +7,10 @@ podman\-logs - Fetch the logs of one or more containers
 **podman** **logs** [*options*] *container* [*container...*]
 
 ## DESCRIPTION
-The podman logs command batch-retrieves whatever logs are present for one or more containers at the time of execution.
-This does not guarantee execution order when combined with podman run (i.e. your run may not have generated
-any logs at the time you execute podman logs
+The podman logs command batch-retrieves whatever logs are present for one or more containers at
+the time of execution. This does not guarantee execution order when combined with podman run.
+Meaning, *run* may not have generated any logs at the time you execute podman logs.  This includes
```
Comment (edsantiago) [inline docs/podman-logs.1.md]: IIUC this isn't the complete picture; in particular, there's the condition where the logs have been generated but immediately discarded.

I've just spent a while looking at `hpcloud/tail` and can't figure out why it's doing this. My expectation is that a reader keeping a file open will see everything written to it even if the file is unlinked.

Comment (cevich) [reply docs/podman-logs.1.md]: @edsantiago thanks for trying, yeah I think this is a fairly complex issue.  [Brent's got some ideas](https://github.com/containers/libpod/issues/2692#issuecomment-474474539) but I doubt anyone knows how long implementation will take.  So update the docs is a stop-gap, something to point to or find and maybe help avoid somebody else going down a deeeeeeeep debugging hole.

Any suggestions for how I could word that better?

**Comment by edsantiago** [comment]
@rhatdan has a descriptive option; see #2719

**Comment by cevich** [comment]
Closing in favor of #2719

**Pull request closed without merging**